PackageExplorer/app.config

```diff
@@ -2,10 +2,70 @@
 <configuration>
     <configSections>
         <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
+            <section name="PackageExplorer.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
             <section name="NuGetPackageExplorer.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false" />
         </sectionGroup>
     </configSections>
     <userSettings>
+        <PackageExplorer.Properties.Settings>
```

Comment:
Oh the settings `GeneratedClassNamespace` got accidentally changed here https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/commit/d9d37f8ff097299b11c0a2e0ffb966d7d14d8d72
@onovotny what is the correct one? `PackageExplorer` or `NuGetPackageExplorer`